src/coreclr/dlls/mscoree/coreclr/CMakeLists.txt

```diff
@@ -167,7 +167,7 @@ if(FEATURE_MERGE_JIT_AND_ENGINE)
     set(CLRJIT_STATIC clrjit_static)
 endif(FEATURE_MERGE_JIT_AND_ENGINE)
 
-if (CLR_CMAKE_TARGET_OSX OR CLR_CMAKE_TARGET_MACCATALYST)
+if (CLR_CMAKE_TARGET_OSX)
```

Comment:
this was erroneously added in https://github.com/akoeplinger/runtime/commit/45887d186c96772cecdc4466af135bcef3885e32, MacCatalyst isn't building this piece.